app/views/jobseekers/registrations/confirm_destroy.html.slim

```diff
@@ -0,0 +1,23 @@
+- content_for :page_title_prefix, t(".page_title")
+
+.govuk-main-wrapper
+  .govuk-grid-row
+    .govuk-grid-column-two-thirds
+      h1.govuk-heading-l = t(".page_title")
+
+      p = t(".page_description_1")
```

Comment:
doesnt really matter tbh but i think using `-` lists in yaml and iterating is better than `_1` `_2` etc. really not a problem though to concern yourself with for some thing this small

Reply:
What do you mean here?